docker/build/installers/install_drivers_deps.sh

```diff
@@ -26,6 +26,9 @@ bash /tmp/installers/install_opencv.sh
 bash /tmp/installers/install_adv_plat.sh "${MY_MODE}"
 bash /tmp/installers/install_proj4.sh
 
+# Required by audio-driver python version
+sudo apt-get install python3-pyaudio
```

Comment:
This line should be written as
```
apt-get -y update && \
      apt-get -y install python3-pyaudio
```

`sudo` is not required when docker build, and we'd better call `apt-get -y update` before `apt-get -y install <package-name>`.  The option `-y` must be specified here to install deb packages non-interactively.

Reply:
Thank you for your explanation! Have modified it and passed test.